db/migrate/20240606152236_create_significant_date_history_reason.rb

```diff
@@ -0,0 +1,11 @@
+class CreateSignificantDateHistoryReason < ActiveRecord::Migration[7.0]
+  def change
+    create_table :significant_date_history_reasons, id: :uuid do |t|
+      t.timestamps
+      t.string :reason_type
```

Comment:
Thinking that, like you said, this might be an enum, but using a string over an integer would be clearer for database poking friends anyway so this is still good.

Might be worth an index on `reason_type` as it could be common to query by type?